main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	service "github.com/Merteg/pl-health-service/pkg"
+	pl_health_service "github.com/Merteg/pl-health-service/proto"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"gopkg.in/mgo.v2/bson"
+
+	"github.com/rs/zerolog/log"
+	"google.golang.org/grpc"
+)
+
+func init() {
+	client, err := mongo.NewClient(options.Client().ApplyURI("mongodb+srv://admin:admin@pl-health-service.s25udti.mongodb.net/test"))
+	if err != nil {
+		fmt.Println(err)
+	}
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
+	err = client.Connect(ctx)
+	if err != nil {
+		fmt.Println(err)
+	}
+	err = client.Ping(ctx, nil)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	targetCollection, err := client.Database("pl-health-service").ListCollectionNames(context.TODO(), bson.M{"name": "target"})
+	if err != nil {
+		fmt.Println(err)
+	}
+	if len(targetCollection) == 0 {
+		_ = client.Database("pl-health-service").CreateCollection(context.TODO(), "target")
+		fmt.Println("target collection created and ready")
+	} else {
+		fmt.Println("target collection ready")
+	}
+	healthCollection, err := client.Database("pl-health-service").ListCollectionNames(context.TODO(), bson.M{"name": "health"})
+	if err != nil {
+		fmt.Println(err)
+	}
+	if len(healthCollection) == 0 {
+		_ = client.Database("pl-health-service").CreateCollection(context.TODO(), "health")
+		fmt.Println("health collection created and ready")
+	} else {
+		fmt.Println("health collection ready")
+	}
+}
+
+func main() {
+	listener, err := net.Listen("tcp", "localhost:8080")
```

Comment:
port should be configurable
you can do it as a constant for now, we will add an ability to override it through something like env vars or yaml later